keras_core/utils/image_dataset_utils.py

```diff
@@ -319,13 +325,12 @@ def image_dataset_from_directory(
             if shuffle:
                 dataset = dataset.shuffle(buffer_size=1024, seed=seed)
 
-        dataset = dataset.prefetch(tf.data.AUTOTUNE)
```

Comment:
Not sure that prefetching should be done before shuffling

Reply:
> Not sure that prefetching should be done before shuffling

so reviously keras used to do local shuffle not global hence I kept'd same.